src/server/api.php

```diff
@@ -991,12 +992,41 @@ function get_covidcast($source, $signals, $time_type, $geo_type, $time_values, $
   } else {
     // fetch most recent issue fast
     $condition_version = '(t.`is_latest_issue` IS TRUE)';
+    $union = False;
   }
   // the query
-  $query = "SELECT {$fields} FROM {$table} {$subquery} WHERE {$conditions} AND ({$condition_version}) ORDER BY {$order}";
+  $query_template = "SELECT {$fields} FROM {$table} {$subquery} WHERE {$conditions} AND ({$condition_version})";
+
+  $from_2021 = str_replace("#TABLE#", "covidcast", $query_template);
+  $query = "";
+  if ($union) {
+    $from_2020 = str_replace("#TABLE#", "covidcast2", $query_template);
+    $query = "({$from_2020}) UNION ({$from_2021})";
```

Comment:
have you considered using an `UNION ALL` instead? Are there any duplicates that need to be filtered out?

Reply:
If there are duplicates that need to be filtered out, wouldn't we want to use `UNION`? If there aren't duplicates I guess `UNION ALL` should be a bit faster?

Or maybe theres a case for not filtering out duplicates through SQL that I'm unaware of

Reply:
There are two kinds of duplicates: 
* rows with is_latest_issue=1 are identical in both tables, except
* some rows with `issue` between 20210201 and 20210205 have different ids and timestamps but identical values 

If we decide to go with this solution and UNION ALL is faster than UNION, we should definitely consider dropping the covidcast2 version of each duplicate.